src/lightning_app/__setup__.py

```diff
@@ -11,7 +11,7 @@
 _SOURCE_ROOT = os.path.join(_PROJECT_ROOT, "src")
 _PACKAGE_ROOT = os.path.join(_SOURCE_ROOT, "lightning_app")
 _PATH_REQUIREMENTS = os.path.join("requirements", "app")
-_FREEZE_REQUIREMENTS = bool(int(os.environ.get("FREEZE_REQUIREMENTS", 0)))
+_FREEZE_REQUIREMENTS = bool(os.environ.get("FREEZE_REQUIREMENTS", 0))
```

Comment:
These changes don't look right: `bool("0") is True`

Reply:
dam, i missed it, thx for catching it :)